tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -4320,60 +4332,61 @@ def closeLoop(self, kernel, tPA, tPB, loopIdx, finalLoop, emitEndLabelOnly=False
       else:
         endCounter = 0
 
-      if kernel["AssertSummationElementMultiple"] % (kernel["DepthU"] * 2) == 0 and endCounter > 0:
-        # if AssertSummationElementMultiple is multiple of DepthU*2, loop exit is necessary only once in 2 Loop iterations
-        #  In endCounter % 2 == 1 case, exit at lc % 2 == 0 (= oddLabel). It means no exit if not oddLabel
```

Comment:
This segment is not related to dot2 kernel, but it leads to runtime error due to undefined varaible **decValue**. Should we modify or remove this?